### DISCUSSION TITLE
Renovate doesn't trigger on my repo.

### DISCUSSION THREAD

**Discussion by xtremerui** [renovatebot/renovate#16857] [Request Help]
Hi, just want to know if there is anyway I can debug or find details about the reason of Renovate bot not trigger on my repo https://github.com/concourse/concourse/blob/master/.github/renovate.json 

When I adding the bot config json I did see the potential PR that it would create when onboarding the bot. But after the scheduled period I haven't seen any PR created by it.

Thank you!

**Comment by rarkins** [comment]
If you log into app.renovatebot.com then do you see logs?

**Comment by xtremerui** [comment]
I do. Didn't know there is a way to check the logs.

It indicates there is indeed a syntax error so the config is invalid. Thanks a lot!
